RHIDP-5999: POST-GA docs release tasks for RHDH release (#1185)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/ref-release-notes-known-issues.adoc

This section lists known issues in {product} {product-version}.

[id="known-issue-rhidp-7529"]
== Incorrect branding in 1.6.0

> Due to a process bug, the upstream branding was used instead of {company-name} branding when releasing the 1.6.0 helm chart. 

Reviewer: ```suggestion
Previously, the upstream branding was used instead of {company-name} branding when releasing the 1.6.0 helm chart. 
```
Not sure we need that much detail.
